dolphin/coordination.py

```diff
@@ -205,3 +218,24 @@ def _synchronized(f, *a, **k):
             return f(*a, **k)
 
     return _synchronized
+
+
+def get_redis_backend_url():
+    password = getattr(CONF.coordination, 'backend_password', None)
+    if password is not None:
+        # If password is needed, the password should be
+        # set in config file with cipher text
+        # And in this scenario, these are also needed for backend:
+        # {backend_type}://[{user}]:{password}@{ip}:{port}.
+        plaintext_password = cryptor.decode(password).decode('utf-8')
```

Comment:
why need two decode here?

Reply:
In order to change the return value of cryptor.decode() from bytes to string
Now modify the cryptor.decode() to return a str, to ensure that when we use a str to decode, what we get after decode should also be a str.